webpack.prod.js

```diff
@@ -0,0 +1,6 @@
+const merge = require('webpack-merge');
+const common = require('./webpack.common.js');
+
+module.exports = merge(common, {
+    mode: 'production',
```

Comment:
I don't see any optimizations for production. Can you add `OptimizeCssAssetsPlugin` and `TerserPlugin` to `optimization` part?